grpc/src/main/java/io/stargate/grpc/service/Service.java

```diff
@@ -742,6 +764,9 @@ private Connection newConnection(AuthenticatedUser user) {
       connection = persistence.newConnection();
     }
     connection.login(user);
+    if (user.token() != null) {
+      connection.clientInfo().ifPresent(c -> c.setAuthenticatedUser(user));
```

Comment:
I know this is not really part of supporting auth in grpc directly, but shouldn't `connection.login()` set the authenticated user of the `ClientInfo` if it's present? It seems strange to have to set it as a separate state.

Reply:
Yeah, I copied what was already in place for CQL, but indeed it seems like this could be done directly in `login()`.